src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -92,7 +92,7 @@
                 <div class="panel panel-default panel-editor last">
                   <div class="panel-heading">
                     <p>{{ 'lbl.Summary'|trans|ucfirst }}
-                      <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpSummary'|trans|ucfirst }}"></abbr>
+                      <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpSummary'|trans|ucfirst }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320